frontend/src/main/components/Sections/SectionsInstructorTable.js

```diff
@@ -0,0 +1,92 @@
+import SectionsInstructorTableBase from "main/components/SectionsInstructorTableBase";
+
+import { yyyyqToQyy } from "main/utils/quarterUtilities.js";
+import {
+  convertToFraction,
+  formatDays,
+  formatInstructors,
+  formatLocation,
+  formatTime,
+  isSection,
+} from "main/utils/sectionUtils.js";
+
+export default function SectionsInstructorTable({ sections }) {
+  // Stryker enable all
```

Comment:
I don't think this comment is necessary; Stryker is enabled by default.
```suggestion
```

Reply:
Resolved!